pkg/pxc/backup/restore.go

```diff
@@ -266,6 +266,8 @@ func AzureRestoreJob(cr *api.PerconaXtraDBClusterRestore, bcp *api.PerconaXtraDB
 		return nil, errors.New("azure storage is not specified")
 	}
 	container, _ := azure.ContainerAndPrefix()
+	destination = strings.TrimPrefix(destination, "azure://"+container+"/")
```

Comment:
it'd be nice to move `azure://` to a constant since we're using it in 5 different places